src/chains-config/metadata-consts/kusama-consts.ts

```diff
@@ -0,0 +1,62 @@
+import { Definition } from '../../types/chains-config';
+
+/**
+ * All Runtimes before v2027
+ */
+const extrinsicBaseWeight = 125000000;
+
+/**
+ * Runtimes after v2026
+ */
+const blockWeights = {
+	baseBlock: 5000000000,
+	maxBlock: 2000000000000,
+	perClass: {
+		normal: {
+			baseExtrinsic: 125000000,
```

Comment:
Just double checking these do not need to be BN or BigInt?
If we convert them to one or the other later maybe best just to store them that way to avoid the downstream conversion